crates/ansi-to-html/src/color.rs

```diff
@@ -114,6 +152,39 @@ pub(crate) enum FourBitColor {
     BrightWhite,
 }
 
+impl FourBitColor {
+    pub(crate) fn is_bright(self) -> bool {
+        matches!(
+            self,
+            Self::BrightBlack
+                | Self::BrightRed
+                | Self::BrightGreen
+                | Self::BrightYellow
+                | Self::BrightBlue
+                | Self::BrightMagenta
+                | Self::BrightCyan
+                | Self::BrightWhite,
+        )
+    }
+
+    pub(crate) fn write_css_var_name(self, s: &mut String) {
```

Comment:
Please rewrite this as a `Display` implementation, so it can be formatted directly.

Reply:
That would involve making another struct to hold all the context needed to display. Is that what you're asking for, or am I missing something?